### ISSUE TITLE
Create Solid app to configure wallet(s) in profile

### ISSUE THREAD

**Issue by rubensworks** [KNowledgeOnWebScale/solid-web-monetization#2]
A small client-side JavaScript app should be created that allows users to log in,
and manage (CRUD) wallets in their profile.

This will be nothing more than a visual interface to add/remove triples in a single RDF document (the profile of the logged in user).

Important here is that the interface is simple and intuitive enough for non-developers.

Depends on #1

Possibly relevant software libraries:

* https://github.com/inrupt/generator-solid-react
* https://github.com/solid/query-ldflex/

**Comment by Falx** [comment]
Added a first React web app with React Router, React Bootstrap and Scss/Typescript enabled.

**It is not doing anything yet, but this allows me to get the ci/cd pipe running, see #3**

**Comment by Falx** [comment]
Login/Logout working for solid oidcProvider https://inrupt.net
Waiting for https://github.com/inrupt/solid-ui-react/pull/276 to get merged into @inrupt/solid-ui-react to enable session restore on refresh.

As testcase, the Wallet page is no reading the /public/card#me dataset
